slug.go

```diff
@@ -376,6 +376,12 @@ func (p *Packer) Unpack(r io.Reader, dst string) error {
 
 		// Get rid of absolute paths.
```

Comment:
This comment got separated from the thing that actually gets rid of absolute paths - I'd move the comment down to line 385.